cloud-bigtable/bigtable.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Initialization action for installing Apache HBase anc configuring it to access
+# cloud-bigtable
+
+# Determine the role of this node
+set -e
+ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+PROJECT=$(/usr/share/google/get_metadata_value ../project/project-id)
+BIGTABLE_PROJECT=$(/usr/share/google/get_metadata_value attributes/bigtable-project || true)
+BIGTABLE_INSTANCE=$(/usr/share/google/get_metadata_value attributes/bigtable-instance || true)
+BIGTABLE_PROJECT=${BIGTABLE_PROJECT:-${PROJECT}}
+MVN_REPO="https://repo1.maven.org/maven2"
+
+# Keep up to date
+BIGTABLE_CLIENT_VERSION='0.9.4'
```

Comment:
Acknowledged. I will try to update frequently.

Unless you keep a latest artifact somewhere, I don't see how this can be avoided.